### ISSUE TITLE
Unexpected key 'WorkGroup' found in params

### ISSUE THREAD

**Issue by zivkaziv** [KoteiIto/node-athena#50]
I'm trying to run the example:

```
var clientConfig = {
    bucketUri: 's3://xxxx'
}

var awsConfig = {
    region: 'xxxx', 
}
 
var athena = require("athena-client")
var client = athena.createClient(clientConfig, awsConfig)

client.execute('SELECT 1').toPromise()
.then(function(data) {
    console.log(data)
})
.catch(function(err) {
    console.error(err)
})
```

But I'm getting the following error:

> Unexpected key 'WorkGroup' found in params

Any idea why?

**Comment by zivkaziv** [comment]
The issue was related to old aws-sdk. Make sure you are using upgraded one

**Comment by KoteiIto** [comment]
Thank you for reporting.I fixed this bug on #52 
Please use v2.5.1.
